app/org/sagebionetworks/bridge/models/studies/ConsentSignature.java

```diff
@@ -5,82 +5,47 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-import org.sagebionetworks.bridge.exceptions.InvalidEntityException;
-import org.sagebionetworks.bridge.json.JsonUtils;
+import org.joda.time.DateTime;
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
 import org.sagebionetworks.bridge.models.BridgeEntity;
 import org.sagebionetworks.bridge.validators.ConsentSignatureValidator;
 import org.sagebionetworks.bridge.validators.Validate;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonFilter;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.ser.impl.SimpleBeanPropertyFilter;
+import com.fasterxml.jackson.databind.ser.impl.SimpleFilterProvider;
 
+@JsonDeserialize(builder=ConsentSignature.Builder.class)
+@JsonFilter("filter")
 public final class ConsentSignature implements BridgeEntity {
 
-    private static final String NAME_FIELD = "name";
-    private static final String BIRTHDATE_FIELD = "birthdate";
-    private static final String IMAGE_DATA_FIELD = "imageData";
-    private static final String IMAGE_MIME_TYPE = "imageMimeType";
+    public static final ObjectWriter SIGNATURE_WRITER = new BridgeObjectMapper().writer(
+            new SimpleFilterProvider().addFilter("filter",
+            SimpleBeanPropertyFilter.serializeAllExcept("signedOn")));
+
     private static final ConsentSignatureValidator VALIDATOR = new ConsentSignatureValidator();
 
     private final @Nonnull String name;
     private final @Nonnull String birthdate;
     private final @Nullable String imageData;
     private final @Nullable String imageMimeType;
+    private final @Nonnull long signedOn;
 
     /** Private constructor. Instances should be constructed using factory methods create() or createFromJson(). */
     @JsonCreator
     private ConsentSignature(@JsonProperty("name") String name, @JsonProperty("birthdate") String birthdate,
-            @JsonProperty("imageData") String imageData, @JsonProperty("imageMimeType") String imageMimeType) {
+            @JsonProperty("imageData") String imageData, @JsonProperty("imageMimeType") String imageMimeType, 
+            @JsonProperty("signedOn") long signedOn) {
```

Comment:
Nitpick: Since Jackson now goe through the Builder, these JsonCreator and JsonProperty annotations now do nothing. We should remove them. (Not a blocker for merging.)